src/stories/Form/index.js

```diff
@@ -0,0 +1,13 @@
+import React from "react";
+
+import Form from "./component";
+
+const StyledForm = (props) => {
+    return (
+        <div>
```

Comment:
Maybe consider taking out the <div>, so that the root element is <Form> ...

Reply:
done as per suggestion